client/header/activity-panel/index.js

```diff
@@ -39,7 +39,7 @@ const HelpPanel = lazy( () =>
 
 const InboxPanel = lazy( () =>
 	import(
-		/* webpackChunkName: "activity-panels-inbox" */ '../../inbox-panel'
+		/* webpackChunkName: "activity-panels-help" */ './panels/inbox/inbox-panel'
```

Comment:
Why change the chunk name here?

Reply:
Nice catch! I modified it by mistake. It was fixed in the commit fe250be